randovania/games/samus_returns/gui/hint_details_tab.py

```diff
@@ -0,0 +1,81 @@
+from __future__ import annotations
+
+import collections
+import random
+from typing import TYPE_CHECKING
+
+from PySide6 import QtWidgets
+
+from randovania.exporter.hints.hint_exporter import HintExporter
+from randovania.games.samus_returns.exporter.hint_namer import MSRHintNamer
+from randovania.gui.game_details.game_details_tab import GameDetailsTab
+from randovania.layout import filtered_database
+from randovania.lib.container_lib import iterate_key_sorted
+
+if TYPE_CHECKING:
+    from randovania.game_description.game_patches import GamePatches
+    from randovania.games.game import RandovaniaGame
+    from randovania.interface_common.players_configuration import PlayersConfiguration
+    from randovania.layout.base.base_configuration import BaseConfiguration
+
+
+class MSRHintDetailsTab(GameDetailsTab):
+    def __init__(self, parent: QtWidgets.QWidget, game: RandovaniaGame):
+        super().__init__(parent, game)
+        self.tree_widget = QtWidgets.QTreeWidget(parent)
+
+    def widget(self) -> QtWidgets.QWidget:
+        return self.tree_widget
+
+    def tab_title(self) -> str:
+        return "Hints"
+
+    def update_content(
```

Comment:
Don't know if we have any tests but also I don't mind it for such a simple non crucial thing.
But: Minimum should be to add the return type here for mypy.